src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  width: 100vw;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  height: 100%;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  list-style-type: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 0;
+  margin: 0;
+  line-height: 60px;
+  margin-left: -20px;
```

Comment:
```suggestion
```
Not to use here this `-` margin you could just remove `margin-left` from your first `.nav-list__item`:

```
.nav-list__item:first-child {
  margin-left: 0;
}
```